extensions/amp-story/1.0/amp-story-reaction.js

```diff
@@ -30,16 +29,17 @@ import {createShadowRootWithStyle} from './utils';
 import {dev} from '../../../src/log';
 import {dict} from '../../../src/utils/object';
 import {getRequestService} from './amp-story-request-service';
-import {htmlFor} from '../../../src/static-template';
-import {toArray} from '../../../src/types';
 
 /** @const {string} */
-const TAG = 'amp-story-quiz';
+const TAG = 'amp-story-reaction';
 
-// TODO(jackbsteinberg): Refactor quiz to extend a general interactive element class
-// and make this an enum on that class.
-/** @const {number} */
-const STORY_REACTION_TYPE_QUIZ = 0;
+/**
+ * @const @enum {number}
+ */
+export const ReactionType = {
+  QUIZ: 0,
+  BINARY_POLL: 1,
```

Comment:
Nit: we can discuss it later, but I think `POLL` will make more sense. Whether it has 2 or 4 elements or images will probably not make any difference to this abstract.

Reply:
Sounds good, I've been thinking about backend implications and since the type is mostly used for analytics, we can have the granularity we want on this enum